### ISSUE TITLE
Names of options in the Link node do not get translated

### ISSUE THREAD

**Issue by masarapmabuhay** [chrisamanse/UsbongKit#10]
Salut! I just noticed that the options in the `Link` node do not get translated, even if the translation is available in the trans folder.

<img width="376" alt="screen shot 2016-10-26 at 4 10 59 pm" src="https://cloud.githubusercontent.com/assets/344066/19719195/734f546a-9b9b-11e6-835e-c0accfc81567.png">

So for example, the name of the option is `The Japanese alphabets`, it doesn't get translated to `Ang Mga Alpabetong Hapon`.

`<task name="imageTextDisplay~Slide5~@audioName=Page8~[I write what Luffy says on my notepad so that everyone can see. If I write in kanji, I would put the hiragana equivalent underneath as well. And if we’re still in Lesson 1 and 2, I would include the ro-maji, which is the romanized alphabet.]~The Japanese alphabets"></task>`

**Comment by chrisamanse** [comment]
Can you test this change if this fixes the issue? 4578d30f535527d370378595abf5a8359e7df7af

I also made the options for `Decision` node translatable. I haven't tested this yet though, but it should work similar to `Link`.

**Comment by masarapmabuhay** [comment]
Salut! I tested your update, https://github.com/usbong/UsbongKit/commit/4578d30f535527d370378595abf5a8359e7df7af, and it worked like magic. C'est magnifique! 

As for the `decision` node, I have yet to test it as well, since I predominantly use `link`. Nonetheless, it would still be good to get `decision` node to work for backward compatibility's sake.

In the meantime, I found that the `radioButtonsWithAnswer` node doesn't get translated as well. Let me create a new issue for this. Merci beaucoup!
